discovery-provider/src/tasks/entity_manager/playlist.py

```diff
@@ -259,10 +267,14 @@ def delete_playlist(params: ManageEntityParameters):
 
 
 def process_playlist_contents(playlist_record, playlist_metadata, block_integer_time):
+    playlist_id = playlist_record.playlist_id
     if playlist_record.metadata_multihash:
         # playlist already has metadata
         metadata_index_time_dict: Dict[int, Dict[int, int]] = defaultdict(dict)
-        for track in playlist_record.playlist_contents["track_ids"]:
+        playlist_tracks = playlist_record.playlist_contents["track_ids"]
+        if (len(playlist_tracks) >= PLAYLIST_TRACK_LIMIT):
```

Comment:
i think this should be in validate or else there might be some unintended things at this point

Reply:
we don't have the full playlist record at the validation step right? validation just covers the incoming message from entity manager